src/js/components/Menu.js

```diff
@@ -149,6 +160,13 @@ var MenuDrop = React.createClass({
     if (this.props.small) {
       classes.push(CLASS_ROOT + "__drop--small");
     }
+    if (this.props.className) {
+      var classNames = this.props.className.split(" ");
+      classNames = classNames.map(function(className) {
+        return className + "__drop";
```

Comment:
I was just wondering if you had a specific instance that demonstrated the need for this? In general, we tend to avoid adding things until we have a specific instance where it is needed, to try to keep the code as simple as possible. I realize we don't always do as good a job of this as we could but I still think it's worth trying.